lib/googleHelpers.js

```diff
@@ -0,0 +1,60 @@
+const { OAuth2Client } = require('google-auth-library')
+const helpers = require('./helpers')
+
+// constants
+const PA_CLIENT_ID = helpers.getEnvVar('PA_CLIENT_ID')
+const PA_CLIENT_SECRET = helpers.getEnvVar('PA_CLIENT_SECRET')
+const PA_GSUITE_DOMAIN = 'brave.coop'
+
+const paOAuth2Client = new OAuth2Client(PA_CLIENT_ID, PA_CLIENT_SECRET, 'postmessage')
+
+async function paGetPayload(idToken) {
+  const ticket = await paOAuth2Client.verifyIdToken({ idToken, audience: PA_CLIENT_ID })
+  return ticket.getPayload()
+}
+
+async function paGetTokens(authCode) {
+  const { tokens } = await paOAuth2Client.getToken(authCode)
+  return tokens
+}
+
+async function paValidateIDToken(idToken) {
+  const payload = await paGetPayload(idToken)
+
+  // return true if:
+  // - the ID token originated from PA
+  // - the account is from the Brave organization (brave.coop)
+  // - the ID token was authenticated by Google
+  // - the experation date hasn't passed
+  return (
+    payload.aud === PA_CLIENT_ID &&
+    payload.hd === PA_GSUITE_DOMAIN &&
+    (payload.iss === 'https://accounts.google.com' || payload.iss === 'accounts.google.com') &&
+    payload.exp > Date.now()
+  )
+}
+
+async function paAuthorize(req, res, next) {
```

Comment:
Are we still using the ClickUp authorization function anywhere? If not, it can be removed.

Reply:
Nope! I just realized that the ClickUp checker is the only function in `clickUpHelpers`, so I've removed it. Thanks for noticing that!